x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvFirstTests.java

```diff
@@ -41,7 +41,7 @@ public static Iterable<Object[]> parameters() {
         cartesianPoints(cases, "mv_first", "MvFirst", DataType.CARTESIAN_POINT, (size, values) -> equalTo(values.findFirst().get()));
         geoShape(cases, "mv_first", "MvFirst", DataType.GEO_SHAPE, (size, values) -> equalTo(values.findFirst().get()));
         cartesianShape(cases, "mv_first", "MvFirst", DataType.CARTESIAN_SHAPE, (size, values) -> equalTo(values.findFirst().get()));
-        return parameterSuppliersFromTypedDataWithDefaultChecks(false, cases);
+        return parameterSuppliersFromTypedDataWithDefaultChecks(false, cases, (v, p) -> "");
```

Comment:
Should this throw? Maybe say something like `all types are valid` or something.

Reply:
I mean, this will fail if a type ends up not being valid.  The function will generate an error message with the string "representable" here, which will not match the test's expected empty string.  Throwing from here doesn't really make that any clearer in my opinion?  But I'm open to discuss.